package.json

```diff
@@ -36,6 +36,11 @@
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start:mockDuo --no-open",
     "test:testcafe-run": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome:headless test/testcafe/spec/ --reporter xunit:build2/reports/junit/testcafe-results.xml",
     "test:testcafe-ci": "run-p -r test:testcafe-setup \"test:testcafe-run\" -- 2>/dev/null",
+    "test:testcafe-en-leaks-setup": "mkdir -p test-reports/testcafe && node scripts/en-leaks-setup-mocks.js && yarn build:dev && yarn start:dev-server --no-open",
```

Comment:
Do we have readMe for these commands ?

Reply:
Not yet, I can add some within the pseudo-loc readme section